### PR TITLE
NO-ISSUE: integrationt tests support for env vars substitution

### PR DESCRIPTION
**Pull request by andfasano**
This patch modifies the integration tests custom comparison functions to allow env vars substitutions into the expected file (in a similar fashion to the standard testscript [cmpenv](https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript) function).

A new custom function `expandFile` has been added to allow en var replacement for the actual files.

In addition, it adds the `RELEASE_IMAGE` env var to the integration tests execution environment referring the current release image pullspec. This will allow to avoid hard-coding the release image pullspec in the integration tests, which is causing a test failure every time the default release image is updated.

Relates to https://github.com/openshift/installer/pull/7874

### CODE REVIEW

File: cmd/openshift-install/agent_internal_integration_test.go [hunk 1]
```diff
@@ -200,6 +210,7 @@ func ignitionStorageContains(ts *testscript.TestScript, neg bool, args []string)
 // [!] isoCmp `isoPath` `isoFile` `expectedFile` check that the content of the file
 // `isoFile` - extracted from the ISO embedded configuration file referenced
 // by `isoPath` - matches the content of the local file `expectedFile`.
+// Environment variables in in `expectedFile` are substituted before the comparison.
```
Comment (patrickdillon) [inline cmd/openshift-install/agent_internal_integration_test.go:213]: nit: 
```suggestion
// Environment variables in `expectedFile` are substituted before the comparison.
```

File: cmd/openshift-install/agent_internal_integration_test.go [hunk 2]
```diff
@@ -267,9 +280,34 @@ func readIgnition(ts *testscript.TestScript, ignPath string) (config igntypes.Co
 	return config, err
 }
 
+// [!] expandFile `file...` can be used to substitute environment variables
+// references for each file specified
```
Comment (patrickdillon) [inline cmd/openshift-install/agent_internal_integration_test.go:284]:
```suggestion
// references for each file specified.
```
linter is complaining about this

**Review by patrickdillon — COMMENTED** [review]
/approve

**Review by andfasano — COMMENTED** [review]
Suggestions reviewed

**Pull request merged**